### ISSUE TITLE
Add the option to remove retention indices before  initializing them from the comment

### ISSUE THREAD

**Issue by hechth** [RECETOX/RIAssigner#120]
This function could be implemented in multiple ways:

- in `init_ri_from_comment` to check whether the keyword for RI type to assign is present or not, and if it is not presend and the RI value is set, then remove it.
- allow cleaning the RI upon reading the data in `_read` in MatchMSData and PandasData
- by adding a `clean_ri` function which sets all RI values in a `Data` object to 0 or None
- not implement anything and just use the setter and set all RI values to None by passing an array of `None`s to the setter

Problem is, that RI values which don't represent actual values are not written by the package, so it is actually not possible to remove RI values with the package. This does not concern the pandas data.
The current implementation of the write function of the MatchMSData class doesn't support writing the value if it is smaller 0.

If we want to allow resetting values etc., we should remove the RI key if the value is 0. The RI values for spectra where there is non numeric information stored is currently set to 0. So this means removing the key from the spectrum in the `_assign_ri_values` function.

TLDR
- [ ] add the else statement to remove the reteion index key if the value is 0
- [ ] implement unit test for this behaviour and make sure that all other tests still pass
